spike.rb

```diff
@@ -0,0 +1,23 @@
+def verse
```

Comment:
I like that you've got this here to show you how far you've come, but normally I wouldn't merge a spike.

Reply:
+1